jta-1_2-standalone/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java

```diff
@@ -50,8 +45,6 @@
 
     private MandatoryCounterManager mandatoryCounterManager;
 
-    private TestEntityRepository testEntityRepository;
-
     private Weld weld;
 
     @BeforeClass
```

Comment:
Well I guess I will have to yield if there is precedent but personally I thought the guidelines suggested otherwise.